scripts/twig_functions.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+# Link contrib twig functions into the components directory
+
+# First we need to find out whether this is inside a Drupal install or standalone Emulsify
+if [ -d ../../../../vendor ]
```

Comment:
I have an older Drupal install that I ran this inside of and it didn't have the /web directory. Should we make this more open-ended to handle that?